src/lib/config.js

```diff
@@ -59,9 +59,7 @@ function getInitialConfig () {
 
 var initialConfig = getInitialConfig()
 
-if (isDefinedConfigValue(process.env.SERVER_NAME)) {
-  initialConfig.serverName = process.env.SERVER_NAME
-}
+initialConfig.serverName = process.env.SERVER_NAME || 'Composr 2.0'
```

Comment:
https://github.com/corbel-platform/corbel-composr/blob/master/src/config/config.json

Is there so it will never be undefined..